silk-core/src/main/kotlin/net/silkmc/silk/core/screen/ScreenExtensions.kt

```diff
@@ -0,0 +1,10 @@
+@file:Suppress("unused")
+
+package net.silkmc.silk.core.screen
+
+import net.minecraft.client.gui.screens.inventory.AbstractContainerScreen
+import net.minecraft.world.inventory.Slot
+import net.silkmc.silk.core.mixin.client.AbstractContainerScreenAccessor
+
+val AbstractContainerScreen<*>.focusedSlot: Slot?
```

Comment:
is there a reason the extension is not called hoveredSlot as well?

Reply:
the reason is I forgored to update it